configs/envoy_double_proxy.template.yaml

```diff
@@ -0,0 +1,110 @@
+---
+{% macro listener(address,ssl,proxy_proto) -%}
+- address: "{{ address }}"
+  {% if ssl -%}
+  ssl_context:
+    alpn_protocols: h2,http/1.1
+    alt_alpn_protocols: http/1.1
+    cert_chain_file: certs/servercert.pem
+    private_key_file: certs/serverkey.pem
+  {% endif -%}
+  {% if proxy_proto -%}
+  use_proxy_proto: true
+  {% endif -%}
+  filters:
```

Comment:
This also hangs off of filter_chains. Take a looks at the listener fields: https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/lds.proto#envoy-api-msg-listener and also the filter_chains field: https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/listener/listener.proto#envoy-api-msg-listener-filterchain